src/main/java/org/opencds/cqf/tooling/modelinfo/ClassInfoBuilder.java

```diff
@@ -1326,14 +1339,21 @@ private ClassInfo buildClassInfo(String modelName, StructureDefinition sd) throw
                 .withIdentifier(sd.getUrl())
                 .withRetrievable(sd.getKind() == StructureDefinitionKind.RESOURCE);
 
+        if (sd.getKind() == StructureDefinitionKind.RESOURCE) {
+            String targetTypeName = getTargetTypeName(sd);
+            if (targetTypeName != null) {
+                info.setTarget(targetTypeName);
+            }
+        }
+
         this.typeInfos.put(qualifiedTypeName, info);
 
         AtomicReference<Integer> index = new AtomicReference<Integer>(1);
         List<ClassInfoElement> elements = new ArrayList<>();
         String path = sd.getType(); // Type is used to navigate the elements, regardless of the baseDefinition
         String id = path; // Id starts with the Type
         // TODO: Switch to differential here, but several of the primitive types declare "value" elements of type string when this happens? (positiveInt, markdown, among others)
```

Comment:
Comment out of date?

Reply:
No, because the TODO still applies, we should be using the Differential, but there's a whole bunch that needs to happen to do that, because it basically means building the full hierarchy, rather than the flattened approach that's taken here.